kafka_utils/kafka_consumer_manager/commands/offset_advance.py

```diff
@@ -88,5 +90,13 @@ def run(cls, args, cluster_config):
                 "with --help option.", file=sys.stderr
             )
             raise
+        except UnknownMemberIdError:
+            print(
+                "Unable to unsubscribe group '{group_name}' from topic '{topic_name}'. \
+                    You must ensure none of the consumers with this consumer group id are running before \
+                    trying to unsubscribe a consumer group with offsets stored in Kafka. Try stopping all \
```

Comment:
Since this command isn't trying to unsubscribe a consumer group, but actually advancing the offset, it might be better to change the printed message a little bit:
```suggestion
                    You must ensure none of the consumers with this consumer group id are running before \
                    trying to advance the offsets stored in Kafka for this consumer group. Try stopping all \
```

Reply:
Good idea, thanks for catching that! I had reused an existing error message from a previous, related issue 😄